whylog_vim/output_formater/output_agregator.py

```diff
@@ -0,0 +1,31 @@
+from functools import partial
+from whylog_vim.consts import Messages
+
+
+class OutputAgregator():
+    def __init__(self):
+        self.buttons = {}
+        self.output_lines = []
+
+    def add(self, element):
+        self.output_lines.append(element)
+
+    def add_commented(self, content):
+        self.add(Messages.PREFIX % content)
+
+    def get_content(self):
+        # The same type as vim buffor.
+        return self.output_lines
+
+    def create_button(self, function, **params):
+        line = self._get_line()
```

Comment:
If in the first we need to create the text of output and then we puts the buttons to the correct lines, we will need to create a structure which will inform where to put which button. I guess this is the clearer solution.